requirements.txt

```diff
@@ -8,3 +8,4 @@ networkx==2.5
 importlib-resources==5.4.0
 typing-extensions==4.1.1
 matplotlib==3.3.3
+multiprocess==0.70.13
```

Comment:
This was needed to allow the use of closures to defer parameter calculation (the stdlib `multiprocessing` fails trying to pickle closures). There might be a way to restructure the code to avoid this.